vm/src/obj/objbyteinner.rs

```diff
@@ -174,42 +319,52 @@ impl PyByteInner {
         elements
     }
 
-    pub fn contains_bytes(&self, other: &PyByteInner, vm: &VirtualMachine) -> PyResult {
+    pub fn contains(&self, needle: PyObjectRef, vm: &VirtualMachine) -> PyResult {
+        match try_as_bytes_like(&needle) {
+            Some(value) => self.contains_bytes(&value, vm),
+            None => match_class!(needle,
+                i @ PyInt => self.contains_int(&i, vm),
+                obj => {Err(vm.new_type_error(format!("a bytes-like object is required, not {}", obj)))}),
+        }
+    }
+
+    fn contains_bytes(&self, other: &[u8], vm: &VirtualMachine) -> PyResult {
```

Comment:
This can return `bool` as there is not exceptions that can be raised

Reply:
not relevant since I merged contains_bytes and contains_int